creusot/tests/should_succeed/ghost/assert_in_ghost.coma

```diff
@@ -0,0 +1,981 @@
+      | s3 = new'0 {_2} (fun (_ret':GhostBox'0.t_ghostbox ()) ->  [ &_0 <- _ret' ] s4)
+      | s4 = bb1 ]
+      
+    | bb1 = return' {_0} ]
+    
+    [ & _0 : GhostBox'0.t_ghostbox () = any_l ()
+    | & _1 : Closure'0.assertinghost_ghostcapture_closure0 = _1
+    | & _2 : () = any_l ()
+    | & y : int32 = any_l () ]
+     [ return' (result:GhostBox'0.t_ghostbox ())-> return' {result} ] 
+end
+module AssertInGhost_GhostCapture
+  let%span sassert_in_ghost0 = "/home/arnaud/Documents/Programmation/Rust/CONTRIBUTIONS/creusot/creusot/tests/should_succeed/ghost/assert_in_ghost.rs" 12 12 12 17
+  
+  let%span span1 = "/home/arnaud/Documents/Programmation/Rust/CONTRIBUTIONS/creusot/creusot-contracts/src/invariant.rs" 8 8 8 12
+  
+  let%span span2 = "/home/arnaud/Documents/Programmation/Rust/CONTRIBUTIONS/creusot/creusot-contracts/src/resolve.rs" 46 8 46 12
+  
+  let%span span3 = "/home/arnaud/Documents/Programmation/Rust/CONTRIBUTIONS/creusot/creusot-contracts/src/ghost.rs" 69 8 69 32
+  
+  let%span span4 = "/home/arnaud/Documents/Programmation/Rust/CONTRIBUTIONS/creusot/creusot-contracts/src/ghost.rs" 77 15 77 16
+  
+  let%span span5 = "/home/arnaud/Documents/Programmation/Rust/CONTRIBUTIONS/creusot/creusot-contracts/src/ghost.rs" 76 14 76 27
+  
+  let%span span6 = "/home/arnaud/Documents/Programmation/Rust/CONTRIBUTIONS/creusot/creusot-contracts/src/ghost.rs" 77 4 77 28
+  
+  let%span span7 = "/home/arnaud/Documents/Programmation/Rust/CONTRIBUTIONS/creusot/creusot/tests/should_succeed/ghost/assert_in_ghost.rs" 16 22 16 32
+  
+  use CreusotContracts_Ghost_GhostBox_Type as GhostBox'0
+  
+  predicate invariant'1 (self : GhostBox'0.t_ghostbox ()) =
+    [%#span1] true
+  
+  predicate inv'1 (_x : GhostBox'0.t_ghostbox ())
+  
+  axiom inv'1 : forall x : GhostBox'0.t_ghostbox () . inv'1 x = true
+  
+  predicate invariant'0 (self : ()) =
+    [%#span1] true
+  
+  predicate inv'0 (_x : ())
+  
+  axiom inv'0 : forall x : () . inv'0 x = true
+  
+  use prelude.prelude.Intrinsic
+  
+  predicate resolve'2 (self : ()) =
+    [%#span2] true
+  
+  use CreusotContracts_Ghost_GhostBox_Type as CreusotContracts_Ghost_GhostBox_Type
+  
+  predicate resolve'0 (self : GhostBox'0.t_ghostbox ()) =
+    [%#span3] resolve'2 (CreusotContracts_Ghost_GhostBox_Type.ghostbox_0 self)
+  
+  let rec new'0 (x:()) (return'  (ret:GhostBox'0.t_ghostbox ()))= {[@expl:precondition] [%#span4] inv'0 x}
+    any
+    [ return' (result:GhostBox'0.t_ghostbox ())-> {[%#span6] inv'1 result}
+      {[%#span5] CreusotContracts_Ghost_GhostBox_Type.ghostbox_0 result = x}
+      (! return' {result}) ]
+    
+  
+  use prelude.prelude.Int32
+  
+  use AssertInGhost_GhostCapture_Closure0_Type as Closure'0
+  
+  predicate resolve'1 [#"/home/arnaud/Documents/Programmation/Rust/CONTRIBUTIONS/creusot/creusot/tests/should_succeed/ghost/assert_in_ghost.rs" 14 4 17 5] (_1 : Closure'0.assertinghost_ghostcapture_closure0)
```

Comment:
huh is it normal that we are storing absolute paths in tests @Armael ?

Reply:
no, that's not expected, of course

Reply:
@arnaudgolfouse How have you generated these files? You should use `cargo test --test ui assert_in_ghost -- --bless`

Reply:
Ah yes, that's my fault, I forgot about this